consensus/mock/consensusDataContainerMock.go

```diff
@@ -191,6 +193,11 @@ func (ccm *ConsensusCoreMock) SingleSigner() crypto.SingleSigner {
 	return ccm.blsSingleSigner
 }
 
+// KeyGenerator -
```

Comment:
L191: ConsensusCore instead ConsensusStore and also in some other files (2 more files)